ui/app/pages/home/home.container.js

```diff
@@ -102,7 +103,7 @@ const mapStateToProps = (state) => {
     isMainnet: getIsMainnet(state),
     originOfCurrentTab,
     shouldShowWeb3ShimUsageNotification,
-    pendingApprovals: Object.values(pendingApprovals),
+    pendingApprovals,
```

Comment:
Nit: it might be worth renaming this prop, now that it no longer includes all pending approvals. Maybe something like `pendingConfirmations`, like you used on the confirmation page.